<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Arbiya-Maitems" name="c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered" tests="8" failures="1" timestamp="25 Jan 2018 13:13:34 GMT" time="25.318" errors="0">
  <testcase name="verifyBillingAddressIsNotNull" time="0.058" classname="c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered"/>
  <testcase name="verifyHeaderAmtIsNotNull" time="0.143" classname="c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered"/>
  <testcase name="verifyOrderIDIsNotNull" time="24.489" classname="c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered"/>
  <testcase name="verifyShippingAddressIsNotNull" time="0.094" classname="c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered"/>
  <testcase name="verifyHeaderModeOfDeliveryIsNotNull" time="0.174" classname="c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered"/>
  <testcase name="verifyCustomerNumIsNotNull" time="0.127" classname="c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered"/>
  <testcase name="verifyHeaderMOPIsNotNull" time="0.106" classname="c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.assertNotEquals(Assert.java:831)
	at org.testng.Assert.assertNotEquals(Assert.java:836)
	at c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered.verifyHeaderMOPIsNotNull(TC_001_VerifyDetailsExistanceInOrderDelivered.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </failure>
  </testcase> <!-- verifyHeaderMOPIsNotNull -->
  <testcase name="verifyCustomerNameIsNotNull" time="0.127" classname="c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered"/>
</testsuite> <!-- c2info_OrderBuk_DeliveredTCs.TC_001_VerifyDetailsExistanceInOrderDelivered -->
